zuul.d/end-to-end.yaml

```diff
@@ -6,6 +6,10 @@
     files:
       - ^ci_framework/roles/.*_build
       - ^ci_framework/roles/build.*
+    irrelevant-files:
+      - ^.*/*.md
+      - ^zuul.d
```

Comment:
Is this intentional, the PR is about skipping doc changes, but this wil skip changes in zuul config, what happens if we change a zuul job variable and the job breaks?

Reply:
I hesitated a bit on that one... Maybe we can keep it, yeah. Probably safer.